android/app/src/main/java/app/organicmaps/routing/NavigationService.java

```diff
@@ -191,7 +198,11 @@ public int onStartCommand(Intent intent, int flags, int startId)
     // re-subscribe for more frequent GPS updates for navigation.
     LocationHelper.from(this).restart();
 
-    return START_STICKY;
+    // Tell the system that this service should not be restarted automatically
```

Comment:
```
Constant to return from [onStartCommand(Intent, int, int)](https://developer.android.com/reference/android/app/Service#onStartCommand(android.content.Intent,%20int,%20int)): if this service's process is killed while it is started (after returning from [onStartCommand(Intent, int, int)](https://developer.android.com/reference/android/app/Service#onStartCommand(android.content.Intent,%20int,%20int))), and there are no new start intents to deliver to it, then take the service out of the started state and don't recreate until a future explicit call to [Context.startService(Intent)](https://developer.android.com/reference/android/content/Context#startService(android.content.Intent)). The service will not receive a [onStartCommand(android.content.Intent, int, int)](https://developer.android.com/reference/android/app/Service#onStartCommand(android.content.Intent,%20int,%20int)) call with a null Intent because it will not be restarted if there are no pending Intents to deliver.
```

We use the primary app process. It doesn't make sense to restart the primary app process without properly initialized framework to start the service.

Reply:
1. "It does not any sense like it's done now, because framework is not initialized" sounds unclear. Does it make sense to enable sticky _and initialize framework properly_ ? What are pros and cons?
2. Does enabled background service permission change the behavior in our favor?

Reply:
> 1. "It does not any sense like it's done now, because framework is not initialized" sounds unclear. Does it make sense to enable sticky _and initialize framework properly_ ? What are pros and cons?
> 
>     2. Does enabled background service permission change the behavior in our favor?

> and initialize framework properly 

Oh, please not. Not in the hotfix.

Reply:
Added comments.